spec/models/ncr/work_order_spec.rb

```diff
@@ -1,4 +1,6 @@
 describe Ncr::WorkOrder do
+  include ProposalSpecHelper
```

Comment:
I think we are already including all spec helpers in all specs with https://github.com/18F/C2/blob/master/spec/rails_helper.rb#L7